Makefile

```diff
@@ -28,7 +28,7 @@ lint:
 	stack exec -- hlint app src test
 
 format:
-	find . -path ./.stack-work -prune -o -path ./archived -prune -o -type f \( -name "*.hs" ! -name "ReplDoc.hs" \) -exec ormolu --mode inplace {} \;
+	find . -path ./.stack-work -prune -o -path ./archived -prune -o -type f -name "*.hs" -exec ormolu --mode inplace {} --ghc-opt -XTypeApplications --ghc-opt -XUnicodeSyntax --ghc-opt -XPatternSynonyms \;
```

Comment:
It seems fine at the moment but do you think it would be worth adding `--ghc-opt -XTemplateHaskell` here as well?

Reply:
According the Tweag that is turned on by default. If we need -XTemplateHaskellQuotes then we'll have to add that there I think. See `ormolu --manual-exts`. Does the relevant code look fine to you?

Reply:
If I'm reading ormolu's code right, `TemplateHaskellQuotes` is enabled by accident, since it's implied by `TemplateHaskell`, and AFAICT ormolu doesn't add `-XNo…` flags for the things that aren't enabled.

Maybe we should explicitly add it, since this seems unintentional and might be changed in future?

Reply:
OK, thanks for point that out. I did notice that it changed a few files' formatting before when I turned on that flag. See if you like the changes in #358.